snuba/migrations/migration.py

```diff
@@ -45,13 +46,64 @@ def backwards(self, context: Context, dry_run: bool) -> None:
         raise NotImplementedError
 
 
-class MultiStepMigration(Migration, ABC):
+class GlobalMigration(Migration, ABC):
+    """
+    Consists of one or more Python functions executed once globally.
```

Comment:
Should these be always data migrations ? 
I am thinking whether we should call them `DataMigration`. That impact the whole cluster and not a specific node (or the query node) at a time.
Otherwise, would you clarify in the comment what we mean by "globally" ?

Reply:
After reading both yours and Evan's comments here it occurred to me that the word global can be rather misleading and we should probably just drop it. I've renamed this class to `CodeMigration`. Although most of the time this is likely to be a data migration, it has the potential to be used for other means too since it's just code.